run_test.py

```diff
@@ -109,7 +115,8 @@ def main():
             'nccl': 'nccl2.4-cuda101',
             'requires': [
                 # TODO(kmaehashi): Remove setuptools version restrictions
-                'setuptools<42', 'pip', 'cython==0.29.13', 'numpy<1.18',
+                'setuptools<42', 'pip', 'cython==0.29.13',
+                'numpy<1.18>={}'.format(numpy_min_version),
```

Comment:
```suggestion
                'numpy<1.18,>={}'.format(numpy_min_version),
```

To confirm it, see the difference between
- `pip install 'numpy<1.18>=1.18'`
- `pip install 'numpy<1.18,>=1.18'`

Reply:
Thanks, force-pushed.